build

```diff
@@ -863,6 +863,8 @@ then
       # strip "-only" suffix from AMPI target
       PROGRAM="${PROGRAM%-only}"
     fi
+    OPTS="$OPTS -DCMK_NO_INTEROP=1 -DCMK_NO_MSG_PRIOS=1 "
+    CONFIG_OPTS="$CONFIG_OPTS --with-prio-type=char "
```

Comment:
Maybe the `$OPTS` and `$CONFIG_OPTS` should be at the end so manually passed parameters can overwrite them.

Reply:
Done